internal/acceptance/secret_scope_test.go

```diff
@@ -58,3 +59,47 @@ func TestAccSecretScopeResource(t *testing.T) {
 		),
 	})
 }
+
+func TestAccSecretScopeResourceAkvWithSp(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	keVaultResourceID := os.Getenv("TEST_KEY_VAULT_RESOURCE_ID")
+	if cloudEnv != "azure" || keVaultResourceID == "" || os.Getenv("") == "ARM_CLIENT_ID" {
+		t.Skipf("not available on %s or don't have key vault and service principal defined", cloudEnv)
+	}
+
+	scope := qa.RandomName("tf-")
+	workspaceLevel(t, step{
+		Template: fmt.Sprintf(`
```

Comment:
```
Template: `
resource "databricks_secret_scope" "my_scope" {
			name = "akv-{var.RANDOM}"
			keyvault_metadata {
				resource_id = "{env.TEST_KEY_VAULT_RESOURCE_ID}"
				dns_name    = "{env.TEST_KEY_VAULT_DNS_NAME}"
			}
		}
`
```

Reply:
Hmmm, If I remove `resourceCheck` then it fails with on non-empty plan check...